### DISCUSSION TITLE
Make scenes area aware

### DISCUSSION THREAD

**Discussion by andreasbrett** [OHF-Voice/intents#2147] [_Ideas]
Hey there,
I want to discuss how scenes should be treated in Assist. IMHO scenes are area/floor specific just like lights. In fact all of my scenes control lights or entities that are in the same room. I have several light scenes for all the rooms in my house. But I can only speak for myself and there might be other use cases for scenes that are not area (or floor) specific.

The problem (finally):
Currently I can only trigger scenes via assist if I specifically name the room they are in ("activate cozy in the living room"). Contrary to light entities, for scenes the area context of the voice satellite is not used (yet). Also light entities **must** be assigned to an area or floor for them to be controlled via Assist. This is not the case for scenes though. I think scenes should also require to be assigned to an area for them to be triggered via Assist. 

What are your opinions on that?

(initiated via https://github.com/home-assistant/intents/pull/2136)

**Comment by tetele** [comment]
I, for one, agree that not just scenes, but every voice command should be context-aware.

Specifically for this purpose, I've created #2142 so that we can better define context-aware tests for context-aware sentences. Once that gets merged, I will do my best to add context-aware sentences to English, which is usually followed by most other languages.

**Reply by andreasbrett** [reply]
Great, glad I'm not not alone.

**Comment by andreasbrett** [comment]
@synesthesiam I know you're deep into other more important stuff. If you can spare some time, please have a look.

**Comment by andreasbrett** [comment]
@synesthesiam Another half year has passed and @frenck decided to close the PR (https://github.com/home-assistant/intents/pull/2136) that was stale since April. I'd really appreciate if this could gain some traction.
